### ISSUE TITLE
Crashing on Server Load [1.6.13]

### ISSUE THREAD

**Issue by WhiteHusky** [raoulvdberge/refinedstorage#2196]
#### Steps to reproduce:
1. Create a fresh server (in this case with the Forge installer)
2. Add RF 1.6.13 to the mods
3. Start the server and watch as it crashes.

#### Version:

- Minecraft: 1.12.2 Server
- Forge: Recommended (1.12.2-14.23.5.2768) & Latest (1.12.2-14.23.5.2823)
- Refined Storage: 1.6.13

[crash-2019-03-23_13.16.28-server.txt](https://github.com/raoulvdberge/refinedstorage/files/2999625/crash-2019-03-23_13.16.28-server.txt)

```net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from Refined Storage (refinedstorage)
Caused by: java.lang.NoClassDefFoundError: net/minecraft/client/renderer/tileentity/TileEntityItemStackRenderer
	at com.raoulvdberge.refinedstorage.RSItems.<clinit>(RSItems.java:15)
	at com.raoulvdberge.refinedstorage.inventory.item.ItemHandlerFilter.<init>(ItemHandlerFilter.java:29)
	at com.raoulvdberge.refinedstorage.tile.grid.portable.TilePortableGrid.<init>(TilePortableGrid.java:127)
	at com.raoulvdberge.refinedstorage.block.info.BlockInfo.createTileEntity(BlockInfo.java:57)
	at com.raoulvdberge.refinedstorage.proxy.ProxyCommon.registerTile(ProxyCommon.java:360)
	at com.raoulvdberge.refinedstorage.proxy.ProxyCommon.registerBlock(ProxyCommon.java:351)
	at com.raoulvdberge.refinedstorage.proxy.ProxyCommon.preInit(ProxyCommon.java:202)
	at com.raoulvdberge.refinedstorage.RS.preInit(RS.java:61)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:624)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:218)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:196)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:135)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:627)
	at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:99)
	at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:333)
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:125)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:486)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.ClassNotFoundException: net.minecraft.client.renderer.tileentity.TileEntityItemStackRenderer
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	... 44 more
Caused by: net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerException: Exception in class transformer net.minecraftforge.fml.common.asm.transformers.SideTransformer@606fc505 from coremod FMLCorePlugin
	at net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerWrapper.transform(ASMTransformerWrapper.java:260)
	at net.minecraft.launchwrapper.LaunchClassLoader.runTransformers(LaunchClassLoader.java:279)
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:176)
	... 46 more
Caused by: java.lang.RuntimeException: Attempted to load class buo for invalid side SERVER
	at net.minecraftforge.fml.common.asm.transformers.SideTransformer.transform(SideTransformer.java:62)
	at net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerWrapper.transform(ASMTransformerWrapper.java:256)
	... 48 more```

**Comment by raoulvdberge** [comment]
Argh! Thx for reporting.

**Comment by raoulvdberge** [comment]
@WhiteHusky  Are you able to test the latest build on Jenkins? Just want an OK before I push to Curse.

https://jenkins.raoulvdberge.com/job/refinedstorage/375/

**Comment by WhiteHusky** [comment]
@raoulvdberge Spooling up a server now...

**Comment by WhiteHusky** [comment]
@raoulvdberge Looks like it no longer crashes on a simple test server, but I haven't went in game and ensure it works propperly. We're going to update our mods and get the server on the latest version of RF.

**Comment by WhiteHusky** [comment]
@raoulvdberge We loaded up our local server and it basic usage seems to work alright. Scheduling crafts, crafting in the grid, pulling out stuff, etc.

**Comment by raoulvdberge** [comment]
Thx for the test. Is now published as 1.6.14 on Curse.
